src/integration-test/java/apptest/mandates/MandatesIT.java

```diff
@@ -1,50 +1,54 @@
 package apptest.mandates;
 
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import org.springframework.http.HttpMethod;
 import org.springframework.http.HttpStatus;
-
+import org.springframework.test.context.jdbc.Sql;
 import se.sundsvall.dept44.test.AbstractAppTest;
 import se.sundsvall.dept44.test.annotation.wiremock.WireMockAppTestSuite;
 import se.sundsvall.myrepresentative.MyRepresentatives;
-
 import apptest.CommonStubs;
 
-@WireMockAppTestSuite(files = "classpath:/Mandates/", classes = {MyRepresentatives.class})
-@Disabled("Needs updated signed data from bolagsverket. Martin is looking into getting new keys to generate this")
+@WireMockAppTestSuite(files = "classpath:/Mandates/", classes = MyRepresentatives.class)
+@Sql({
+    "/db/scriptsIT/truncate.sql",
+    "/db/scriptsIT/testdata.sql"
+})
 class MandatesIT extends AbstractAppTest {
 
     @Test
     void test1_getMandates_shouldReturnCompleteResponse() {
```

Comment:
Rename to `test001_getMandates_shouldReturnCompleteResponse()` (for test folder sorting purposes)?